ui/app/pages/routes/index.js

```diff
@@ -96,6 +105,15 @@ class Routes extends Component {
         })
       }
     })
+
+    getActiveTab()
+  }
+
+  componentDidUpdate (prevProps) {
+    const { addressConnectedToCurrentTab, showAccountDetail } = this.props
+    if (addressConnectedToCurrentTab && addressConnectedToCurrentTab !== prevProps.addressConnectedToCurrentTab) {
+      showAccountDetail(addressConnectedToCurrentTab)
```

Comment:
Sorry I'm not _quite_ following what's going on here - maybe just getting a bit sleepy :sweat_smile: 

What scenario is this meant to handle?

Reply:
If the user has a connected origin open in a tab, and then opens the extension via the popup view (i.e. clicks the fox in the top right of the browser) the selected address should change to the account connected to the origin of the current tab.